code-challenges.test.js

```diff
@@ -57,3 +133,17 @@ const vowelTester3 = "challenges"
 // Expected output: 2
 
 // b) Create the function that makes the test pass.
+
+const indexOfFirstVowel = (string) => {
+    for (let i=0; i<string.length; i++) {
+      if (string[i] === "a" || string[i] === "e" || string[i] === "i" ||     string[i] === "o" || string[i] === "u") {
+            return [i]   
```

Comment:
By itself, i represents the index, no need for square brackets when referencing the index.  It is only when you are accessing the value that you need to use square brackets with i in it.